CONTRIBUTING.md

```diff
@@ -16,6 +16,8 @@ You can do this online, and it only takes a minute.
 
 2. Understand [coding agreements](packages/README.md).
 
+3. **If you work at Google:** and don't have [DCM](https://dcm.dev/) enabled, [enable and activate it](#enabling-and-activating-dcm).
```

Comment:
Can we switch this to:

```
**[Optional]** Follow instructions to [enable and activate DCM (Dart Code Metrics)](#enabling-and-activating-dcm).
```

Since contributors who don't work at Google should still be able to use DCM if they want to.

Reply:
Ah, I see your point here, sure.

Reply:
Done ✅